cloud-operator/templates/rolebinding.yaml

```
@@ -26,4 +26,18 @@ subjects:
 roleRef:
   kind: Role
   name: hubble-client-certs-reader
-  apiGroup: rbac.authorization.k8s.io
\ No newline at end of file
+  apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: illumio-cloud-operator-secret-reader-binding
+  namespace: gke-managed-dpv2-observability
+subjects:
+- kind: ServiceAccount
+  name: {{ .Values.serviceAccount.name }}
+  namespace: {{ .Values.defaultNamespace | default .Release.Namespace }}
```

Why would we ever refer to the namespace?
```suggestion
  namespace: {{ .Release.Namespace }}
```
This RoleBinding should be identical to the RoleBinding created in `kube-system` above, except for the namespace.